package.json

```diff
@@ -98,6 +98,7 @@
     "@testing-library/react": "10.4.9",
     "@testing-library/react-hooks": "5.0.3",
     "@types/copy-webpack-plugin": "6.4.0",
+    "@types/enhanced-resolve": "3.0.7",
```

Comment:
can we also add `enhanced-resolve` as devDependency to root ? 

- the pattern is to have every dep via single version policy and locally only when its required to run.
- this will also provide better context (like why are these types installed here without real runtime version )

Reply:
Done f3f5c20.